requirements-testing.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# All requirements for running PerfKitBenchmarker unit tests.
+-rrequirements.txt
+-rrequirements-alicloud.txt
```

Comment:
These are the only requirements files required to run unit tests - everything else installs a command-line tool or conditionally-imported module. I'll make a note.